{
    "URI.MalformedURIException.html": {
        "modified": "2021-07-24 18:49:52.778073",
        "size": "10.52KB",
        "md5": "3954438dd83b9e7b04e340966db6d3ad",
        "sha256": "970e02dfaddfdbf7b0efa59d38ca2a00343ac07a85a159ffdf2a41a5b8cd8c78"
    },
    "Metadata": {
        "Content-Encoding": "ISO-8859-1",
        "Content-Type": "text/html; charset=ISO-8859-1",
        "X-Parsed-By": [
            "org.apache.tika.parser.DefaultParser",
            "org.apache.tika.parser.html.HtmlParser"
        ],
        "X-TIKA:content_handler": "ToTextContentHandler",
        "X-TIKA:embedded_depth": "0",
        "X-TIKA:parse_time_millis": "2",
        "dc:title": "URI.MalformedURIException (Axis API)",
        "keywords": "org.apache.axis.types.URI.MalformedURIException class",
        "resourceName": "b'URI.MalformedURIException.html'",
        "title": "URI.MalformedURIException (Axis API)"
    }
}